src/main/java/uk/gov/hmcts/reform/coh/controller/validators/Validator.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.coh.controller.validators;
+
```

Comment:
This is just an interface that each Validation Enum will implement. the "test" method is the thing that will do the validation